File: Programming/calculator.py
## Program of menu based calculator
import os
def add(num1, num2):
    return(num1 + num2)
def subtract(num1, num2):
    return(num1 - num2)
def multiply(num1, num2):
    return(num1 * num2)
def divide(num1, num2):
    return(num1/num2)

while 1:
    os.system('clear')
    print("This is  menu for calculator")
    print("Enter 1 for addition")
    print("Enter 2 for subtraction")
    print("Enter 3 for multiplication")
    print("Enter 4 for division")
    print("Enter 0 for exit")
    option = int(input("Enter your choice: "))
    if option == 0:
        break
    if option > 4 or option < 0:
        print("You have entered wrong choice")
        continue
    num1 = int(input("Enter first number: "))
    num2 = int(input("Enter second number: "))
    result = 0
    if option == 1:
        result = add(num1, num2)
        print("Result of addition is ",result)
    elif option ==  2:
        result = subtract(num1, num2)
        print("Result of subtrction is  ",result)
    elif option == 3:
        result = multiply(num1, num2)
        print("Result of multiplication is ",result)
    elif option == 4:
        result = divide(num1, num2)
        print("Result of division is ",result)
    else:
        print("You have entered wrong choice")
    input("Press enter to continue: ")



    

   

File: Programming/calculator.java
class Calculator{
    public static int addition(int a,  int b){
        return a + b;
    }
    public static int subtraction(int a,  int b){
        return a - b;
    }
    public static int multiplication(int a,  int b){
        return a * b;
    }
    public static int division(int a,  int b){
        return a / b;
    }
    public static void main(String[] args){
        System.out.println("Addition of two numbers " +addition(2,3));
        System.out.println("Subtraction of two numbers " +subtraction(5,3));
        System.out.println("Multiplication of two numbers " +multiplication(2,3));
        System.out.println("Division of two numbers " +division(6,3));
    }
}